include/gz/physics/FrameSemantics.hh

```diff
@@ -156,6 +156,15 @@ namespace gz
         /// \return A FrameID object that corresponds to _identity.
         protected: virtual FrameID GenerateFrameID(
             const Identity &_identity) const;
+
+        /// \brief An implementation class can use this function to get the
+        /// interface that _frameID holds onto in its reference-counted pointer.
+        /// \tparam T The stored pointer is cast to this type.
+        /// \param[in] _frameID the frame identifier to cast an interface from
+        /// \return A raw pointer from the stored shared_ptr but cast to the
+        ///         /// provided type T
```

Comment:
is the extra `///` intentional?

Reply:
Nope, that was just my IDE trying to be helpful. Fixed by [0550d56](https://github.com/gazebosim/gz-physics/pull/395/commits/0550d566e4be8b598a0d91330c106f523cbaaa19)